docs/en/observability/synthetics-private-location.asciidoc

```diff
@@ -1,5 +1,24 @@
 [[synthetics-private-location]]
-= Add a private location
+= Monitor a private endpoint
+
+To monitor resources on private networks use {{agent}} to create a Private Location, or allow access to your private endpoints to the Synthetics Service.
+Private Locations via Elastic Agent require only outbound connections from your network, while allowing access to the Synthetics Service requires allowing
+inbound access, thus posing an additional risk that users must assess.
```

Comment:
This gives no reason why you _would_ choose to allow the Synthetics service to access a private endpoint, it only warns you _against_ doing it. When _should_ you use it?

>thus posing an additional risk that users must assess

Also, who does `users` refer to here?

Reply:
Good point, you'd do it because you don't want to run infrastructure yourself, will add that